finetune/adapter_v2.py

```diff
@@ -214,81 +209,6 @@ def train(
             save_adapter_v2_checkpoint(fabric, model, checkpoint_path)
 
 
-@torch.no_grad()
```

Comment:
This refactor seems unnecessary for the PR. Could it be discussed? I don't think it is a good idea to import loops from other scripts. IMO the duplication is a feature, not an inconvenience, because it allows someone to copy paste the entire script and everything needed is there.

Reply:
I can undo it and duplicate the `measured_flops` function.

Note that there's a precedent for this: the generation loop is copied across scripts.

And it's really a simple change, the code is exactly the same, it's not like its abstracting different code into a single class/function.

I'll revert it here anyways